Dockerfile.conduit

```diff
@@ -0,0 +1,2 @@
+FROM public.ecr.aws/docker/library/postgres:latest
+ENTRYPOINT ["tail", "-f", "/dev/null"]
```

Comment:
What is the purpose of this? This can be removed as we will be building an image and pushing to the ECR.

```suggestion

```